Makefile

```diff
@@ -99,6 +107,44 @@ crystal: $(O)/crystal ## Build the compiler
 deps: $(DEPS) ## Build dependencies
 llvm_ext: $(LLVM_EXT_OBJ)
 
+.PHONY: install
+install: $(O)/crystal man/crystal.1.gz ## Install the compiler at DESTDIR
+	$(INSTALL) -D -m 0755 "$(O)/crystal" "$(BINDIR)/crystal"
+
+	$(INSTALL) -d -m 0755 $(DATADIR)
+	cp -av src "$(DATADIR)/src"
+	rm -rf "$(DATADIR)/$(LLVM_EXT_OBJ)" # Don't install llvm_ext.o
+	cp -av samples "$(DATADIR)/examples"
```

Comment:
Should this be part of the docs target perhaps?

Reply:
Yeah, that might be better. I used the current distribution-scripts as role model and the samples are part of the program tarball. But I agree it would make more sense to place them in the docs package.
I also don't really understand why it's renamed from `samples` to `examples`. Should be one or the other.